report.md

```diff
@@ -2,9 +2,47 @@
 
 ## Subsystem 1 (MFA)
 
-## Subsystem 2 (Interface for Therapists & Patients)
-Interface will be written in HTML, PHP and SQL to allow database access. Google reCAPTCHA will be included to prevent bots from trying to mass access the system. Each unique user will have their own unique userID and password for login. Therapists and patients will have different login privileges as follow.
+---
+
+Subsystems 2 to 4 will support the following functionalities with the following parameters:
+
+1. Registration
+    1. IC

```

Comment:
maybe just assume prof does not know anything about ic. spell out the whole thing as National Registration Identification Card (NRIC) ba hahah